timer.py

```diff
@@ -0,0 +1,23 @@
+import timeit
+
+
+def timer(f, *args, **kwargs):
+    """
+    function to measure the time that function takes.

```

Comment:
What function? please describe more :)
And start a sentence with a capital letter.